### ISSUE TITLE
workspace init: METS schemaLocation is wrong

### ISSUE THREAD

**Issue by bertsky** [OCR-D/core#694]
The current namespace location for the METS namespace `http://www.loc.gov/METS/` is set to `http://www.loc.gov/standards/mets/version17/mets.v1-7.xsd`, which is very much out of date (from 2008, current version is 1.12.1). I don't know if we rely on any newer features, but downstream applications like DFG Viewer or long-time preservation archive ingest might require more recent versions.

Since METS updates seem to keep the namespace name, and the web server provides a fixed location for the current XSD, I suggest using `http://www.loc.gov/standards/mets/mets.xsd` as schemaLocation.

* ocrd.workspace.cli: workspace_init
* ocrd.resolver: workspace_from_nothing
* ocrd_models.ocrd_mets: empty_mets
* ocrd_models.constants: METS_XML_EMPTY
* ocrd_models: mets-empty.xml

**Comment by bertsky** [comment]
BTW, the current 1.12.1 is also already included in `ocrd_validators/mets.xsd`.

**Comment by kba** [comment]
Yes, the METS should be at the newest version, we just never noticed the `schemaLocation` is wrong until you and @erikmitk looked :)
